experiments/cmd/benchtab/main.go

```diff
@@ -34,7 +35,7 @@ func main() {
 	cfg.Password = config.password
 
 	if !config.dontPrepare {
-		initSession, err := scylla.NewSession(cfg)
+		initSession, err := scylla.NewSession(context.Background(), cfg)
```

Comment:
How about defining a context using `signal.Notify(context.Background(), ...)` and passing it everywhere instead passing uninterruptable context?